### DISCUSSION TITLE
Half screen problem in iBAX wallet.

### DISCUSSION THREAD

**Discussion by madasretisireh** [IBAX-io/go-ibax#1065] [Bug Submission]
### Clear and concise description of the problem

When we want to send using the iBAX wallet on the phone, the screen appears halfway. It is necessary to turn the phone screen. This situation must be corrected. Thanks.

### Suggested solution

The sending box can be fully displayed on the screen.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment]
Weaver is an IDE tool for developers and we are cooperate with JUTKEY team.

Please refer to this discussion: #629
